src/ensembl/src/content/app/browser/zmenu/zmenu-types.ts

```diff
@@ -25,6 +25,7 @@ export enum Markup {
 export enum ZmenuAction {
   CREATE = 'create_zmenu',
   DESTROY = 'destroy_zmenu',
+  ACTIVITY_OUTSIDE = 'zmenu-activity-outside', // TODO: sometime later, unify underscores vs hyphens (together with Genome Browser)
```

Comment:
Can this be `CLICK_OUTSIDE` to make it more specific?

Reply:
@msmanoj I think `CLICK_OUTSIDE` would be too specific. And in case we start supporting touch screens, it will need to be changed back to something more generic for outside activity.

Reply:
Fair point. But even if we support touch screen, I think onClicks will still be called for touch events. My only concern is that `ACTIVITY_OUTSIDE` sounds like any activity that happens outside. For example, even moving the pointer could be an activity we perform outside zmenu but we don't close the zmenu for that.

Reply:
The actual message string sent to genome browser contains the word "activity". I would prefer the constant to reflect that.

_(But we hope to have some kind of auditing of the messages passed between genome browser and browser chrome in the future)_